client/ayon_nuke/plugins/load/load_camera.py

```diff
@@ -196,3 +204,52 @@ def remove(self, container):
         node = container["node"]
         with viewer_update_and_undo_stop():
             nuke.delete(node)
+
+
+class AlembicCameraLoader(_NukeCameraLoader):
+    """
+    This will load alembic camera into script.
+    """
+    extensions = {"abc"}
+    label = "Load Alembic Camera"
+    enabled = True
```

Comment:
```suggestion
class AlembicCameraLoader(_NukeCameraLoader):
    """
    This will load alembic camera into script.
    """
    extensions = {"abc"}
    label = "Load Alembic Camera"
    enabled = True
```
I dont think it's necessary to create AlembicCameraLoader inherited from _NukeCameraLoader, we can just let the AlembicCameraLoader becoming the base class of the camera loader. Something like:
```python
class CameraLoader(load.LoaderPlugin):
    """
    This will load alembic camera into script.
    """
    extensions = {"abc"}
    label = "Load Alembic Camera"
    enabled = True

    def load(self, context, name, namespace, data):
        # your code

class FbxCameraLoader(CameraLoader):
    """
    This will load fbx camera into script.
    """
    extensions = {"fbx"}
    label = "Load FBX Camera"

    def load(self, context, name, namespace, data):
        # your code
```

Reply:
@jakubjezek001, want to hear your thought on this. 
Do you think that it's good to keep the code this way or we can optimize the code as I suggested?

Reply:
From my POV - I understand @robin-ynput's point - however, if we go down that route please move out that abstract class from the file, move it somewhere into another file and do not import it into this file but import the file that contains it.

✅ 
```python
from x.y import z

class Plugin(z.CameraLoader)
```
Instead of:

🔴 
```python
from x.y.z import CameraLoader

class Plugin(CameraLoader)
```

Just so that `pyblish` plugin discovery does not even find that class and ignores it completely. It's just cleaner from the perspective of Pyblish not loading it (and hence it also not appearing in the "Skipped Plugins" list in publisher UI, etc.)

I'm fine with either way of the proposed ways - but just wanted to note this.

Reply:
That makes perfect sense. 
In that case I'll go ahead and implement initial @moonyuet suggestion and merge this one, just to ensure this goes out today. (if someone want to change that after, I'll do the work no problem).